src/main/java/com/courtcanva/ccfranchise/repositories/OrderRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.courtcanva.ccfranchise.repositories;
+
+import com.courtcanva.ccfranchise.models.Order;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+
+public interface OrderRepository extends JpaRepository<Order, Long> {
+    List<Order> findByIdIn(List<Long> ids);
```

Comment:
should find by ID and order status

Reply:
My operation is after you have found by status, so maybe I don't need to find by status again

Reply:
I got your ticket wrong.
Your method should be correct.